test/DssVestTopUp.test.ts

```diff
@@ -157,10 +158,15 @@ describe("DssVestTopUp", function () {
       });
 
       it("should vest accrued tokens", async function () {
-        await topUp.refundUpkeep();
-        const vested = await token.balanceOf(topUp.address);
-
-        expect(vested).to.eq(maxDepositAmt);
+        const abi = ["event Vest(uint256 indexed id, uint256 amt)"];
```

Comment:
Why not use `event VestedTokensWithdrawn` from the tested contract and avoid the extra steps to decode?